### PR TITLE
feat: cron service

### PR DESCRIPTION
**Pull request by alecthomas**
Functional but not not completely wired up. Tests pass, metrics aren't hooked up, no main service binary.

### CODE REVIEW

File: backend/cron/service.go [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+			if err != nil {
+				logger.Errorf(err, "Failed to calculate next run time")
+				continue
+			}
+			job.next = nextRun
+			cronQueue[0] = job
+			orderQueue(cronQueue)
+
+			if err := callCronJob(ctx, verbClient, job); err != nil {
+				logger.Errorf(err, "Failed to execute cron job")
```
Comment (matt2e) [inline backend/cron/service.go:112]: Looks like we're dropping support for retry policies on cron? We should probably make that a validation check to ensure no retry directives on cron job verbs if so.

**Review by matt2e — APPROVED** [review]

**Comment by matt2e** [comment]
Looks sweet and relatively simple 👍

File: backend/cron/service.go (hunk 1 above)
Comment (matt2e) [reply backend/cron/service.go:112]: Or maybe just a ticket to add retry logic back in

Comment (alecthomas) [reply backend/cron/service.go:112]: It seems likely we'll be moving retries to Istio.

**Pull request merged**